README.md

```diff
@@ -580,6 +582,93 @@ The view function here is only an example. In practice thunks are only
 relevant if you are rendering a complicated view that takes
 significant computational time to generate.
 
+## JSX
+
+[JSX](https://facebook.github.io/jsx/) is a xml-like syntax extension to ecmascript.
```

Comment:
XML? ECMAScript?

Reply:
That's right. See https://facebook.github.io/jsx/

Reply:
I meant case of letters. `xml` should be `XML`. `ecmascript` is `ECMAScript` indeed.